applications/openshift/general/resource_requests_limits_in_daemonset/test/has_resource_requests_Limits.pass.sh

```diff
@@ -0,0 +1,268 @@
+                                        "name": "varlog"
+                                    },
+                                    {
+                                        "mountPath": "/var/lib/docker/containers",
+                                        "name": "varlibdockercontainers",
+                                        "readOnly": true
+                                    }
+                                ]
+                            }
+                        ],
+                        "dnsPolicy": "ClusterFirst",
+                        "restartPolicy": "Always",
+                        "schedulerName": "default-scheduler",
+                        "securityContext": {},
+                        "terminationGracePeriodSeconds": 30,
+                        "tolerations": [
+                            {
+                                "effect": "NoSchedule",
+                                "key": "node-role.kubernetes.io/master",
+                                "operator": "Exists"
+                            }
+                        ],
+                        "volumes": [
+                            {
+                                "hostPath": {
+                                    "path": "/var/log",
+                                    "type": ""
+                                },
+                                "name": "varlog"
+                            },
+                            {
+                                "hostPath": {
+                                    "path": "/var/lib/docker/containers",
+                                    "type": ""
+                                },
+                                "name": "varlibdockercontainers"
+                            }
+                        ]
+                    }
+                },
+                "updateStrategy": {
+                    "rollingUpdate": {
+                        "maxSurge": 0,
+                        "maxUnavailable": 1
+                    },
+                    "type": "RollingUpdate"
+                }
+            },
+            "status": {
+                "currentNumberScheduled": 6,
+                "desiredNumberScheduled": 6,
+                "numberAvailable": 6,
+                "numberMisscheduled": 0,
+                "numberReady": 6,
+                "observedGeneration": 1,
+                "updatedNumberScheduled": 6
+            }
+        }
+    ],
+    "kind": "List",
+    "metadata": {
+        "resourceVersion": "",
+        "selfLink": ""
+    }
+}
+EOF
+
+
+jq_filter='[ .items[] | select(.metadata.namespace | startswith("kube-") or startswith("openshift-") | not) | select( .spec.template.spec.containers[].resources.requests.cpu == null  or  .spec.template.spec.containers[].resources.requests.memory == null or .spec.template.spec.containers[].resources.limits.cpu == null  or  .spec.template.spec.containers[].resources.limits.memory == null )  | .metadata.name ]'
```

Comment:
@Connect2naga - correct me if I'm wrong should this just skip as the namespace starts with 'kube-' i.e kube-system) even if we have resource/limits set ?

Reply:
yes Luigi, it will skip these two namespaces.

Reply:
yes this scenario's we will cover in unit test.

Reply:
added unit test case for namespace skip